engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,52 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return Math.round(pow(d,0.55) * 8.45f);
+    }
+    
+    private float getInteractionRange(float amount){
+        double d = (double)amount;
+        return Math.round(pow(d,0.6) * 3.9f);
+    }
+
+    private float getRunFactor(float amount){
+       return Math.round(5.5f + (6.4f * amount) - (0.06f * amount * amount)) / 10f;
+    }
```

Comment:
Maybe use a similar formular to the ones you use above? Is there any reason this should be quadratic?
```suggestion
    private float getRunFactor(float height){
       return Math.round(Math.pow(height/DEFAULT_HEIGHT, 0.5) * DEFAULT_RUN_FACTOR);;
    }
```

Reply:
well....its just that i painstakingly constructing the other 2 functions manually,before i realized that it is much easier to arrive at them through regression,and that is how i acquired the third function.I must admit @skaldarnar you are really thorough